### PR TITLE
aliza: init at 1.29.2

### PR DESCRIPTION
**Pull request by Mounium**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/science/medicine/aliza/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    wrapProgram $out/bin/aliza \
+      --prefix LD_LIBRARY_PATH : ${stdenv.lib.makeLibraryPath [ qt4 zlib stdenv.cc.cc ]}
+
+    wrapProgram $out/bin/aliza-vtkvol \
+      --prefix LD_LIBRARY_PATH : ${stdenv.lib.makeLibraryPath [ qt4 zlib stdenv.cc.cc libX11 libXt ]}
+  '';
+
+  meta = {
+    homepage = "http://www.aliza-dicom-viewer.com";
```
Comment (Mic92) [inline pkgs/applications/science/medicine/aliza/default.nix]: Can you also add a license and a description? Do you want to maintain this package?

Comment (Mounium) [inline pkgs/applications/science/medicine/aliza/default.nix]: I couldn't really make out what may be the license based on the website: http://www.aliza-dicom-viewer.com/info/copyright
As for the other problems, I'm adding those.

Comment (Mic92) [inline pkgs/applications/science/medicine/aliza/default.nix]: It is `licenses.unfreeRedistributable` as they only provide binaries.

Comment (Mounium) [inline pkgs/applications/science/medicine/aliza/default.nix]: Thank you!

Comment (Mounium) [inline pkgs/applications/science/medicine/aliza/default.nix]: Should something be done about it being hosted on google drive?

Comment (Mic92) [inline pkgs/applications/science/medicine/aliza/default.nix]: Does they keep old releases (is the URL stable)? Apart from that it is not a problem.

File: pkgs/applications/science/medicine/aliza/default.nix [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    ${patchelf}/bin/patchelf \
+      --interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
+      $out/bin/aliza-vtkvol
+
+    wrapProgram $out/bin/aliza \
+      --prefix LD_LIBRARY_PATH : ${stdenv.lib.makeLibraryPath [ qt4 zlib stdenv.cc.cc ]}
+
+    wrapProgram $out/bin/aliza-vtkvol \
+      --prefix LD_LIBRARY_PATH : ${stdenv.lib.makeLibraryPath [ qt4 zlib stdenv.cc.cc libX11 libXt ]}
```
Comment (Mic92) [inline pkgs/applications/science/medicine/aliza/default.nix]: You can use `--set-rpath` of patchelf instead of `LD_LIBRARY_PATH` to set the library path.

Comment (Mounium) [inline pkgs/applications/science/medicine/aliza/default.nix]: Tried, but it results in the following error message multiple times: 
`./result/bin/aliza: ./result/bin/aliza: no version information available (required by ./result/bin/aliza)`

File: pkgs/applications/science/medicine/aliza/default.nix (hunk 1 above)
Comment (Mounium) [inline pkgs/applications/science/medicine/aliza/default.nix]: Unfortunately I couldn't find old releases on the website

**Comment by Mic92** [comment]
Could you ask upstream to keep old releases. Otherwise the url will likely break in future.

**Comment by Mounium** [comment]
@Mic92 They've answered my email, unfortunately they won't host older releases, but they allowed me to do so. So I will host the last versions on my Google Drive account adding newer ones as I bump the nixos package, if it's okay with you.

**Comment by Mic92** [comment]
That is ok.

**Comment by Mounium** [comment]
@Mic92 I've added the note, and at the same time changed the package from deb to rpm-based, as that was recommended upstream. Also, bumped to next version which resulted in more dependencies, so I reformatted the wrapProgram part a bit.

**Comment by Mic92** [comment]
ok, please make sure to never delete it from your drive.

**Pull request merged**